lib/rpcc_dpdk/rte_flow_ndn.c

```diff
@@ -910,3 +910,133 @@ rpc_rte_flow_indir_action_conf_destroy(rcf_rpc_server *rpcs,
 
     RETVAL_VOID(rte_flow_indir_action_conf_destroy);
 }
+
+rpc_rte_flow_action_handle_p
+rpc_rte_flow_action_handle_create(rcf_rpc_server *rpcs, uint16_t port_id,
+                                  rpc_rte_flow_indir_action_conf_p conf,
+                                  rpc_rte_flow_action_p action,
+                                  struct tarpc_rte_flow_error *error)
+{
+    tarpc_rte_flow_action_handle_create_in   in = {};
+    tarpc_rte_flow_action_handle_create_out  out = {};
+    te_log_buf                              *tlbp;
+
+    in.port_id = port_id;
+    in.conf = (tarpc_rte_flow_indir_action_conf)conf;
+    in.action = (tarpc_rte_flow_action)action;
+
+    rcf_rpc_call(rpcs, "rte_flow_action_handle_create", &in, &out);
+
+    CHECK_RETVAL_VAR_RPC_PTR(rte_flow_action_handle_create, out.handle);
+
+    tlbp = te_log_buf_alloc();
+    TAPI_RPC_LOG(rpcs, rte_flow_action_handle_create,
+                 "port_id=%" PRIu16 ", conf=" RPC_PTR_FMT ", "
```

Comment:
IMHO it is very unfriendly to log indirect action configuration as a pointer. It should be decoded and logged properly.